projects/observability/src/pages/apis/service-detail/instrumentation/service-instrumentation.component.ts

```diff
@@ -24,9 +35,24 @@ import { map } from 'rxjs/operators';
   `
 })
 export class ServiceInstrumentationComponent {
-  public serviceName$: Observable<string | undefined> = this.breadcrumbsService.breadcrumbs$.pipe(
-    map(breadcrumbs => (breadcrumbs.length > 0 ? breadcrumbs[breadcrumbs.length - 1]?.label : ''))
-  );
+  public serviceScoreSubject: BehaviorSubject<ServiceScoreResponse | undefined> = new BehaviorSubject<
+    ServiceScoreResponse | undefined
+  >(undefined);
+
+  public orgScoreResponse$: Observable<OrgScoreResponse>;
+
+  public constructor(
+    private readonly breadcrumbsService: BreadcrumbsService,
+    private readonly serviceInstrumentationService: ServiceInstrumentationService
+  ) {
+    this.breadcrumbsService.breadcrumbs$
+      .pipe(map(breadcrumbs => (breadcrumbs.length > 0 ? breadcrumbs[breadcrumbs.length - 1]?.label : undefined)))
+      .subscribe(serviceName => {
+        this.serviceInstrumentationService
+          .getServiceScore(serviceName!)
+          .subscribe(serviceScore => this.serviceScoreSubject.next(serviceScore));
```

Comment:
Have created a generic service to get the service name, will raise a PR for it soon.

Reply:
That'll be great!